exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,6 @@
-const name = " Daniel  ";
+var firstName = " Daniel  ";
+var nameTrim = firstName.trim();
+var firstNameLength = firstName.length;
+var message = "My name is " + nameTrim + " and my name is " + firstNameLength + " characters long."
 
 console.log(message);
```

Comment:
this code did not meet the requirement because you're getting the wrong name length